config.toml.example

```diff
@@ -628,6 +628,11 @@ changelog-seen = 2
 # If an explicit setting is given, it will be used for all parts of the codebase.
 #new-symbol-mangling = true|false (see comment)
 
+# Select LTO mode that will be used for compiling rustc. By default, thin local LTO (LTO within a
+# single crate) is used. You can also select "thin" or "fat" to apply Thin/Fat LTO on the
```

Comment:
```suggestion
# single crate) is used (like for any Rust crate). You can also select "thin" or "fat" to apply Thin/Fat LTO on the
```

May also want to be re-wrapped. I'm not 100% sure this is true, but that's my recollection.